sumatra/datastore/filesystem.py

```diff
@@ -24,12 +24,13 @@ class DataFile(DataItem):
     def __init__(self, path, store, creation=None):
         self.path = path
         self.full_path = os.path.join(store.root, path)
-        if os.path.exists(self.full_path):
-            stats = os.stat(self.full_path)
-            self.size = stats.st_size
-        else:
-            raise IOError("File %s does not exist" % self.full_path)
-            # self.size = None
+        if not os.path.exists(self.full_path):                          # check if data in absolute path
+            if os.path.exists(os.path.join('Data', path)):              # else check in relative path of Data folder (portable)
```

Comment:
It is an backup for finding data. Mostly, the file in full_path exists, but if not, it checks in Data folder.